tests/Finder/Finder.basic.phpt

```diff
@@ -20,6 +20,12 @@ function export($iterator)
 }
 
 
+test(function() { // count the results
+	$finder = Finder::findFiles('file.txt')->in('files');
+	Assert::same(1, sizeof($finder));
```

Comment:
there is the `count` method in the `Assert` class. https://github.com/nette/tester/blob/master/Tester/Framework/Assert.php#L230